cli-tests/src/test.rs

```diff
@@ -252,14 +249,14 @@ async fn test_command_succeeds_with_bundle_using_bep() {
     .failure();
 
     let requests = state.requests.lock().unwrap().clone();
-    assert_eq!(requests.len(), 4);
-
-    let tar_extract_directory = assert_matches!(&requests[2], RequestPayload::S3Upload(d) => d);
+    assert_eq!(requests.len(), 3);
+    assert_matches!(requests[0], RequestPayload::CreateBundleUpload(_));
+    let tar_extract_directory = assert_matches!(&requests[1], RequestPayload::S3Upload(d) => d);
+    assert_matches!(requests[2], RequestPayload::TelemetryUploadMetrics(_));
```

Comment:
When there's only 3-4 requests, it makes sense to assert each of them. In this case I was confused about what changed and it would be nice to have a historical reference of the changes in the requests themselves.